app/(flickz)/layout.tsx

```diff
@@ -34,6 +35,7 @@ export default async function FlickzLayout({ children }: DashboardLayoutProps) {
         </div>
       </header>
       <main className="overflow-hidden px-4">{children}</main>
+      <Toaster />
```

Comment:
It will be good to add toaster in the global layout.tsx (app/layout.tsx) so we can use it in website and we are also using it while signup
![image](https://github.com/flickzapp/flickz/assets/62984427/98454163-7b37-4e92-8e83-eb2f61b17e1a)